File: module/Radio/src/Radio/Entity/Scheduling.php
<?php

namespace Radio\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity 
 * @ORM\Table(name="scheduling")
 * */
class Scheduling {

    /**
     * @ORM\Id 
     * @ORM\Column(type="integer") 
     * @ORM\GeneratedValue 
     * */
    protected $id;

    /**
     * @ORM\Column(type="smallint") 
     * */
    protected $weekType;

    /**
     * @ORM\Column(type="smallint") 
     * */
    protected $weekDay;

    /**
     * @ORM\Column(type="smallint") 
     * */
    protected $hourFrom;

    /**
     * @ORM\Column(type="smallint") 
     * */
    protected $minFrom;

    /**
     * @ORM\Column(type="smallint") 
     * */
    protected $duration;   
    /**
     * @ORM\Column(type="date") 
     * */
    protected $validFrom;

    /**
     * @ORM\Column(type="date") 
     * */
    protected $validTo;

    /**
     * @ORM\ManyToOne(targetEntity = "Show", inversedBy = "schedulings")
     * @ORM\JoinColumn(name = "radioshow_id", referencedColumnName = "id")
     */
    protected $show;

    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        $this->id = $id;
    }

    public function getWeekType() {
        return $this->weekType;
    }

    public function setWeekType($weekType) {
        $this->weekType = $weekType;
    }

    public function getWeekDay() {
        return $this->weekDay;
    }

    public function setWeekDay($weekDay) {
        $this->weekDay = $weekDay;
    }

    public function getHourFrom() {
        return $this->hourFrom;
    }

    public function setHourFrom($hourFrom) {
        $this->hourFrom = $hourFrom;
    }

    public function getMinFrom() {
        return $this->minFrom;
    }

    public function setMinFrom($minFrom) {
        $this->minFrom = $minFrom;
    }
  
    public function getValidFrom() {
        return $this->validFrom;
    }

    public function setValidFrom($validFrom) {
        $this->validFrom = $validFrom;
    }

    public function getValidTo() {
        return $this->validTo;
    }

    public function setValidTo($validTo) {
        $this->validTo = $validTo;
    }

    public function getShow() {
        return $this->show;
    }
    public function getDuration() {
        return $this->duration;
    }

    public function setDuration($duration) {
        $this->duration = $duration;
    }

    public function toArray() {
        $a = array();
        $a['weekType'] = $this->getWeekType();
        $a['weekDay'] = $this->getWeekDay();
        $a['hourFrom'] = $this->getHourFrom();
        $a['minFrom'] = $this->getMinFrom();
        $a['duration'] = $this->getDuration();    
        return $a;
    }
    
    public function setShow($show) {
        $this->show = $show;
    }



}

File: util/model/src/main/java/hu/radio/tilos/model/Scheduling.java
package hu.radio.tilos.model;

import javax.persistence.*;
import java.sql.Timestamp;
import java.util.Date;

@Entity
@Table(name = "scheduling")
public class Scheduling {

    @Id
    private int id;

    @Basic
    int weekDay;

    @Basic
    int hourFrom;

    @Basic
    int minFrom;

    @Basic
    int duration;


    @Temporal(TemporalType.TIMESTAMP)
    @Column
    private Date validFrom;

    @Temporal(TemporalType.TIMESTAMP)
    @Column
    private Date validTo;

    @Temporal(TemporalType.TIMESTAMP)
    @Column
    private Date base;

    @Basic
    private int weekType;

    @ManyToOne()
    @JoinColumn(name = "radioshow_id", referencedColumnName = "id")
    Show show;

    public int getWeekDay() {
        return weekDay;
    }

    public void setWeekDay(int weekDay) {
        this.weekDay = weekDay;
    }

    public int getHourFrom() {
        return hourFrom;
    }

    public void setHourFrom(int hourFrom) {
        this.hourFrom = hourFrom;
    }

    public int getMinFrom() {
        return minFrom;
    }

    public void setMinFrom(int minFrom) {
        this.minFrom = minFrom;
    }

    public int getDuration() {
        return duration;
    }

    public void setDuration(int duration) {
        this.duration = duration;
    }

    public Date getValidFrom() {
        return validFrom;
    }

    public void setValidFrom(Date validFrom) {
        this.validFrom = validFrom;
    }

    public Date getValidTo() {
        return validTo;
    }

    public void setValidTo(Date validTo) {
        this.validTo = validTo;
    }

    public Date getBase() {
        return base;
    }

    public void setBase(Date base) {
        this.base = base;
    }

    public int getWeekType() {
        return weekType;
    }

    public void setWeekType(int weekType) {
        this.weekType = weekType;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Show getShow() {
        return show;
    }

    public void setShow(Show show) {
        this.show = show;
    }
}
